src/signer/signer.py

```diff
@@ -253,13 +253,13 @@ def run_signer(wallet_id=None, key_pair_seed=None, nonce_seed=None):
 
         elif user_input.lower() == SignerCommands.SEND_PUBLIC_KEY.value:
             logging.info("Generating and posting the public key...")
-            handle_create_xpub(wallet, relay_manager, nostr_private_key)
-
+            handle_create_xpub(wallet, relay_manager, private_key)
```

Comment:
```suggestion
            handle_create_xpub(wallet, relay_manager, nostr_private_key)
```

I think this should be kept `nostr_private_key`? I renamed it as part of this cleanup a little over a week ago https://github.com/0xBEEFCAF3/munstr/pull/18